STM32F405RGTx/Core/Src/rtos_threads/state_machine.cpp

```diff
@@ -518,7 +518,7 @@ void StateMachineThread::runStateMachine(void *argument) {
             Error_Handler();
     }
 	// CurrentState = (*StateMachineThread::SM[CurrentState].Event)();
-    sendCANHeartbeat();
+    // sendCANHeartbeat();
```

Comment:
What's the reason you removed the can heartbeat

Reply:
debugging thing I forgot to uncomment, I'll fix before merging